frontend/src/navigation/Header.js

```diff
@@ -33,15 +46,34 @@ export default function Header() {
   };
 
   return (
-    <AppBar elevation={0} position="static">
-      <Toolbar className={classes.root}>
-        <BackButton className={classes.icon} />
-        <IconButton onClick={handleOpenHome}>
-          <h2 className={classes.title}>hatschi</h2>
-        </IconButton>
-        <IconButton onClick={handleOpenProfile}>
-          <PersonRoundedIcon className={classes.icon} />
-        </IconButton>
+    <AppBar elevation={0} position="static" className={classes.root}>
+      <Toolbar className={classes.toolbar}>
+        <Grid
+          container
+          direction="row"
+          justify="space-between"
+          alignItems="center"
+          display="grid"
```

Comment:
do you have to set `display="grid"` when already using the `Grid`component from material UI?